Universal_Robots_Client_Library/package.xml

```diff
@@ -1,14 +1,9 @@
 <?xml version="1.0"?>
-<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>ur_client_library</name>
   <version>0.1.1</version>
```

Comment:
Perhaps we should set the version to 0.0.0 for now

Reply:
Done.  Thanks!